examples/hooks/App.css

```diff
@@ -31,7 +31,7 @@ body {
 .Search-header {
   display: grid;
   align-items: flex-start;
-  grid-template-columns: 1fr auto;
+  grid-template-columns: 1fr auto auto;
```

Comment:
this changes the style slightly so we can fit in the HitsPerPage widge

<img width="1030" alt="Screenshot 2021-11-22 at 16 37 44" src="https://user-images.githubusercontent.com/2982512/142890351-897065b7-da7e-493d-9d9e-f659e14ff89c.png">
t